src/main/java/com/playtika/automation/school/test/framework/clients/ServiceFeignClient.java

```diff
@@ -0,0 +1,38 @@
+package com.playtika.automation.school.test.framework.clients;
+
+import com.playtika.automation.school.test.framework.pojo.nested.Note;
+import com.playtika.automation.school.test.framework.pojo.requests.CreateNoteRequest;
+import com.playtika.automation.school.test.framework.pojo.requests.UpdateNoteRequest;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@FeignClient(
+        name = "notes-client",
+        url = "https://taschool-notes-service.herokuapp.com",
```

Comment:
should be placed in application properties

Reply:
fixed